charts/mcp-trino/values-production.yaml

```
@@ -0,0 +1,139 @@
+        - protocol: TCP
+          port: 443   # HTTPS
+        - protocol: TCP
+          port: 53    # DNS
+        - protocol: UDP
+          port: 53    # DNS
+
+# Production affinity rules
+affinity:
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - podAffinityTerm:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)